src/c++/perf_analyzer/genai-perf/genai_perf/main.py

```diff
@@ -30,17 +30,33 @@
 import traceback
 from argparse import Namespace
 from pathlib import Path
+from typing import TYPE_CHECKING
 
 import genai_perf.logging as logging
-from genai_perf import parser
-from genai_perf.constants import DEFAULT_PARQUET_FILE
 from genai_perf.exceptions import GenAIPerfException
-from genai_perf.export_data.json_exporter import JsonExporter
-from genai_perf.llm_inputs.llm_inputs import LlmInputs
-from genai_perf.llm_metrics import LLMProfileDataParser
-from genai_perf.plots.plot_config_parser import PlotConfigParser
-from genai_perf.plots.plot_manager import PlotManager
-from genai_perf.tokenizer import Tokenizer, get_tokenizer
+
+# Import heavy modules to make type checker happy
+if TYPE_CHECKING:
```

Comment:
This if statement is used so that these imports only occur for static checkers.

I tried using the function import_heavy_modules in here to avoid code duplication and that works for pylance, but mypy complains. From research and experimentation, this seems the most straightforward and concise way with our current linters.